nengo_viz/static/viz.js

```diff
@@ -378,6 +378,12 @@ VIZ.DataStore.prototype.get_shown_data = function() {
     return shown;
 }
 
+VIZ.DataStore.prototype.is_at_end = function() {
+    var ts = this.sim.time_slider;
+    return (ts.last_time < ts.first_shown_time + ts.shown_time + 0.01);
```

Comment:
Hmm... I'm a bit confused by this logic. I'm guessing that 0.01 is a fudge factor for float comparison; in that case, shouldn't it be something like
```javascript
ts.last_time >= ts.first_shown_time + ts.shown_time - 0.01
```
?